osutil/inotify/README.md

```diff
@@ -0,0 +1 @@
+These files were downloaded from https://github.com/kubernetes/utils/tree/master/inotify
```

Comment:
I think we should try to include this as a git-subtree, similar what we did with `osutil/udev`. IIRC we discussed this but I couldn't find the discussion, was there a reason to not do it? (or is it done already and I'm just not seeing it in the GH ui?)

Reply:
At first I misunderstood it. But now there are some changes done, respect to the original code, suggested by James.

Reply:
I've been searching, and it seems to not be a good solution for this. All that I found was, basically, to create a new repo from that folder, and then do the subtree. That preserves the history, but, if I got it right, won't keep it updated.

The only solution that comes to my mind is to do a full subtree of that repo, and link the files we need into the build tree... What do you think?

Reply:
Ok, as I commented in mattermost, I found that fsnotify.v0 is compatible with Go 1.13, so it should solve this problem. I uploaded a commit with the changes.

Reply:
Ok, as commented in mattermost, doing a subtree from a folder is quite hacky, so we decided to revert that last commit with the change to fsnotify.v0 and, instead, just keep the code here and manage it ourselves.